src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -41,8 +41,12 @@ class ContentExplorer extends Component {
         contentExplorerMode: ContentExplorerModePropType.isRequired,
         /** Props that contains the custom search input. Is rendered in header actions */
         customInput: PropTypes.func,
+        /** Whether the user can see the breadcrumbs represented with the folder tree button */
+        hasFolderTreeBreadcrumbs: PropTypes.bool,
         /** Any extra items in the header to the right of the search input (and new folder button) */
         headerActionsAccessory: PropTypes.node,
+        /** Whether the user can see the Include Subfolders toggle with breadcrumb updated design */
+        canIncludeSubfolders: PropTypes.bool,
```

Comment:
same comment about removing the `with breadcrumb updated design`. can also alphasort this